src/rustllvm/ArchiveWrapper.cpp

```diff
@@ -71,7 +79,12 @@ extern "C" RustArchiveIterator*
 LLVMRustArchiveIteratorNew(RustArchive *ra) {
     Archive *ar = ra->getBinary();
     RustArchiveIterator *rai = new RustArchiveIterator();
+#if LLVM_VERSION_MINOR <= 8
     rai->cur = ar->child_begin();
+#else
+    Error err;
+    rai->cur = ar->child_begin(err);
```

Comment:
The `err` `Error` value needs to be checked, otherwise this happens with debug LLVM:
```
Program aborted due to an unhandled Error:
Error value was Success. (Note: Success values must still be checked prior to being destroyed).
```